src/getEntityTypescriptPostgres.ts

```diff
@@ -258,66 +255,39 @@ async function getEntityTypescriptPostgres(
 			table,
 		});
 
-		// const custom_decorators: Record<string, string> = {
-		// 	"@Fields.string": "@KitFields.string#@kitql/remult",
-		// };
-
-		let decorator = decorator_infered;
-		for (const custom in custom_decorators) {
-			if (decorator === custom) {
-				decorator = custom_decorators[custom] ?? "THIS_CANNOT_HAPPEN";
-				break;
-			}
-		}
+		const decorator = customDecorators[decoratorInfered] ?? decoratorInfered;
 
 		// TODO: extract this logic from the process column
 		await handleEnums(enums, dataType, provider, udtName);
 
-		if (!defaultOrderBy && ["order", "name", "nom"].includes(columnName)) {
+		if (
+			!defaultOrderBy &&
+			["order", "name", "nom", "username"].includes(columnName)
```

Comment:
why "name", "nom", "username" ?

Reply:
It's good default default in my case.

I always keep in mind that it's a one shoot generator.
When my entities will be good, and my app running with remult, I will never use the cli anymore.

Cli is good to Kickstart a project when you have a db. After, I don't the need